source/js/buyers-guide/homepage-c-slider.js

```diff
@@ -118,21 +124,39 @@ export default {
       }
     );
 
+    /**
+     * querySelectors in this event listener are constantly polled on scroll
+     * to ensure when a browser width changes creepo-face still works as expected
+     * And getElementById is used in places to speed up DOM searching
+     */
     window.addEventListener(
       "scroll",
       () => {
-        if (window.innerWidth < 768) {
-          const element = document.getElementById("product-list");
-          const position = element.getBoundingClientRect();
-          /**
-           * Check if the product grid area is partially visible in the viewport
-           */
-          if (position.top < window.innerHeight && position.bottom >= 0) {
-            STICKY_BAR.classList.add("search-active");
-          } else {
-            // Product area is no longer in the viewport at all.
-            STICKY_BAR.classList.remove("search-active");
-          }
+        const element = document.getElementById("product-list");
+        const position = element.getBoundingClientRect();
+
+        // If on desktop, don't delay moving creepo-face into the corner
+        // If on mobile, make the creepy face move to the corner sooner
+        const offset = window.innerWidth > 768 ? window.innerHeight - CATEGORY_BAR.offsetHeight : 100;
+
+        /**
+         * Check if the product grid area is partially visible in the viewport
+         */
+        if (
+          position.top + offset < window.innerHeight &&
+          position.bottom >= 0
+        ) {
+          SEARCH_BAR.classList.add("search-active" ,"creep-o-meter-moved");
+        }
+
+        const recommendProduct = document.getElementById("recommend-product");
+        const heightFromTop = recommendProduct.getBoundingClientRect().top;
```

Comment:
Let's check if `document.getElementById("recommend-product");` exist before we do anything more to it.